game/scripts/npc/items/item_bloodthorn.txt

```diff
@@ -65,14 +65,14 @@
     //-------------------------------------------------------------------------------------------------------------
     "AbilityValues"
     {
-      "bonus_intellect"                                   "20 30 45 65 90"
+      "bonus_intellect"                                   "20 25 30 35 40"
```

Comment:
Why make INT scaling shit? I am not a fan. Put 20/25/35/50/70.

Reply:
seemed really weird considering how much else bloodthorn gives now. its not really an int item anymore and nothing like this scaled anywhere near as well.